### PR TITLE
docs: Add catalog application chart bundle to install instructions

### PR DESCRIPTION
**Pull request by cbuto**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-83643

## Description of changes being made

This PR adds steps to the airgap kommander installation to include downloading/installing the DKP catalog application chart bundle. 

I only included how to upload the charts during the installation. It can also be done post installation using the [upload command](https://dev-docs.d2iq.com/dkp/kommander/2.2/install/air-gapped/#kommanders-internal-helm-repository)

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/install/air-gapped/index.md [hunk 1]
```diff
@@ -50,27 +50,27 @@ Optionally, specify the output using the `-o` parameter:
...
+  kommander helmmirror upload chart [chartTarball]
+  ```
 
 Or upload a new bundle:
 
-    ```bash
-    kommander helmmirror upload bundle [chartsTarball]...
-    ```
+  ```bash
+  kommander helmmirror upload bundle [chartsTarball]...
```
Comment (cwyl02) [inline pages/dkp/kommander/2.2/install/air-gapped/index.md:72]: do we need `...` here?

File: pages/dkp/kommander/2.2/install/air-gapped/index.md [hunk 2]
```diff
@@ -50,27 +50,27 @@ Optionally, specify the output using the `-o` parameter:
...
+  ```
 
 Please check the built-in help text for each command for more information.
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/install/air-gapped/index.md:75]:
```suggestion
Check the built-in help text for each command for more information.
```

File: pages/dkp/kommander/2.2/install/air-gapped/index.md [hunk 3]
```diff
@@ -281,13 +281,30 @@ Based on the network latency between the environment of script execution and the
 1. Download the Kommander charts bundle:
 
     ```bash
-    wget "https://downloads.mesosphere.com/dkp/dkp-kommander-charts-bundle_${VERSION}.tar.gz
+    wget "https://downloads.mesosphere.com/dkp/dkp-kommander-charts-bundle_${VERSION}.tar.gz"
+    ```
+
+1. (Optional) Download the [DKP catalog applications][dkp_catalog_applications] chart bundle if you intend on deploying any DKP catalog applications:
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/install/air-gapped/index.md:287]:
```suggestion
1. Download the [DKP catalog applications][dkp_catalog_applications] chart bundle if you intend on deploying any DKP catalog applications:
```

Comment (cmastr) [reply pages/dkp/kommander/2.2/install/air-gapped/index.md:287]: You already told them 'if you intend' so this optional isn't needed.

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.2/install/air-gapped/index.md (hunk 1 above)
Comment (cbuto) [reply pages/dkp/kommander/2.2/install/air-gapped/index.md]: I don't think so, removed in 6301b1243106ff388d71c6e805f04f8e7c508d90!

File: pages/dkp/kommander/2.2/install/air-gapped/index.md (hunk 2 above)
Comment (cbuto) [reply pages/dkp/kommander/2.2/install/air-gapped/index.md]: Fixed in 6301b1243106ff388d71c6e805f04f8e7c508d90

File: pages/dkp/kommander/2.2/install/air-gapped/index.md [hunk 4]
```diff
@@ -281,13 +281,30 @@ Based on the network latency between the environment of script execution and the
...
 1. To install Kommander in your air-gapped environment using the above configuration file, enter the following command:
 
     ```bash
-    kommander install --installer-config ./install.yaml --kommander-applications-repository kommander-applications_${VERSION}.tar.gz --charts-bundle dkp-kommander-charts-bundle_${VERSION}.tar.gz
+    kommander install --installer-config ./install.yaml \
+    --kommander-applications-repository kommander-applications_${VERSION}.tar.gz \
+    --charts-bundle dkp-kommander-charts-bundle_${VERSION}.tar.gz
+    ```
+
+    To upload the optional [DKP catalog applications][dkp_catalog_applications] chart bundle, add an additional `--chart-bundle` flag to the `install` command:
```
[the remark this answers is not in the document]
Comment (cbuto) [reply pages/dkp/kommander/2.2/install/air-gapped/index.md]: fixed in 6301b1243106ff388d71c6e805f04f8e7c508d90

**Review by cwyl02 — APPROVED** [review]

**Review by takirala — APPROVED** [review]

**Pull request merged**